packages/core/src/components/GradientScroller/story.tsx

```diff
@@ -70,6 +71,18 @@ withLeftAndRightArrows.story = {
   name: 'With left and right arrows.',
 };
 
+export function withArrowsButNotShown() {
+  return (
+    <GradientScroller showArrows hideScrollbar>
+      <Text>{"Hello"}</Text>
```

Comment:
`{"Hello"}` is throwing some prettier/lint errors. update this with `<LoremIpsum short />` please (we try to use that for story copy)